### PR TITLE
Rename "Add Signature" to "Add More Signatures"

### LINKED ISSUE #242: Rename "Add Signature" to "Add more signatures"
### Problem

Add signature confuses users when there are no new items in the to do section. Re-sign might be more intuitive for the user to understand that the transaction was already previously signed.

### Solution

See above

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by statop**
**Description**:
Rename "Add Signature" to "Submit more signatures"

**Related issue(s)**:

Fixes #242

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by statop** [comment]
- [ ] confirm with @lbaird

**Comment by lbaird** [comment]
I had asked that it remain “Add signature” rather than “Re-sign”.  Let’s
discuss this before making the change.

File: tools-integration/src/test/java/com/hedera/hashgraph/client/integration/BatchTransactionEndToEndTest.java [hunk 1]
```diff
@@ -208,10 +208,10 @@ public void signBatchTransaction_test() throws Exception {
 		Node addSignature;
 
 		do {
-			addSignature = find("ADD SIGNATURE");
+			addSignature = find("SUBMIT MORE SIGNATURES");
```
Comment (DSMatusevich) [inline tools-integration/src/test/java/com/hedera/hashgraph/client/integration/BatchTransactionEndToEndTest.java:211]: Have you checked that the button size is not going too much out of proportion?

**Review by DSMatusevich — DISMISSED** [review]

**Review by statop — COMMENTED** [review]
wait for discussion

**Review by DSMatusevich — APPROVED** [review]

**Pull request merged**